### ISSUE TITLE
Request to share pre-trained model of PFE

### ISSUE THREAD

**Issue by vinits5** [Ontheway361/pfe-pytorch#3]
Hi, Thanks for this great work of pfe-pytorch. I would be grateful to you if you can share the pre-trained model and some training logs. I am getting a negative loss value during the training of pfe model on my custom dataset. Could you also help me with this problem? I am also reachable on my email (vinitsarode5@gmail.com).

**Comment by Ontheway361** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    U can ask the f**k author of the paper.
    
        
            
        
    
    
      
    
        
                    
            
                
                    
                            
                                
                            
                            
                                2459699580
                            
                    
                        
                            
                                    ***@***.***
                            
                        
                
            
        
        
    
 
    

  
    ---- Replied Message ----
  
  
    
      
        
         From 
        
        
            ViNiT ***@***.***>
            
        
      
      
        
         Date 
        
        
        10/29/2022 11:03
        
      
      
        
         To 
        
        
         
          
            ***@***.***>
            
          
        
      
      
        
         Cc 
        
        
          
            ***@***.***>
            
          
        
      
      
        
         Subject 
        
        
              [Ontheway361/pfe-pytorch] Request to share pre-trained model of PFE (Issue #3)
        
      
    
  
  
Hi, Thanks for this great work of pfe-pytorch. I would be grateful to you if you can share the pre-trained model and some training logs. I am getting a negative loss value during the training of pfe model on my custom dataset. Could you also help me with this problem? I am also reachable on my email ***@***.***).

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
